src/components/common/withoutHeader.ts

```diff
@@ -0,0 +1 @@
+export const withoutHeader = () => null;
```

Comment:
This is not a component, this is more of a navigation configuration. Move this to a separate file inside navigation folder. Create a file called navigationOptions and move this function there.

Reply:
Also, rework this function to export const withoutHeader = () => ({ headerShown: false });
This way you will not have to create an object inside navigationOptions prop of screens.